### ISSUE TITLE
FFS evaluation

### ISSUE THREAD

**Issue by yizenghan** [Vchitect/Latte#103]
Hi, I used your released ffs_b_2.pt to generate 2048 videos (they seem normal), converted them to frames, and calculated FVD with the folder "preprocess_ffs/test/images". The outcome FVD is quite low （>100) and unstable (large variance). Did I do anything wrong?

**Comment by yizenghan** [comment]
If I use train/images for real data path, it becomes ~84.

**Comment by maxin-cn** [comment]
There seems to be nothing wrong with the whole process.

**Comment by yizenghan** [comment]
If so, the FVD itself could be so unstable? Which data path should we select to evaluate models? It seems that the training set contains more samples for more stable evaluation. On the test set, my tested FVD lags behind your reported values by a large margin.

**Comment by maxin-cn** [comment]
> There seems to be nothing wrong with the whole process.

Sorry for this. You need to use the train split to evaluate FVD. Whichever split you trained on should be used as real data.

**Comment by yizenghan** [comment]
Thanks, I got it

**Comment by smile2game** [comment]
I get the same result the the outcome FVD is quite low （>100) and unstable (large variance). I don't know how to solve this problem.How did you solve it?
